lib/elixir/pages/anti-patterns/code-anti-patterns.md

```diff
@@ -359,6 +366,20 @@ iex> Graphics.plot(bad_point)
 
 The behavior above is unexpected because our function should not work with points without a `:x` key. This leads to subtle bugs, as we may now pass `nil` to another function, instead of raising early on.
 
+Here's another example showing how the non-assertive map access can propagate through the system:
+
+```elixir
+iex> point_without_x = %{y: 10}
+%{y: 10}
+iex> {x, y, _} = Graphics.plot(point_without_x)
+{nil, 10, nil}
+iex> distance_from_origin = :math.sqrt(x * x + y * y)
+** (ArithmeticError) bad argument in arithmetic expression
+    :erlang.*(nil, nil)
```

Comment:
This is a neat example!
I wonder if it can be compacted with the block above with `bad_point` since it is very close.